pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtlOptions.java

```diff
@@ -162,15 +162,16 @@ public interface FhirEtlOptions extends PipelineOptions {
 
   void setRowGroupSizeForParquetFiles(int value);
 
+  // TODO: Either remove this feature or properly implement patient history fetching based on
+  //   Patient Compartment definition.
   @Description(
       "The active period with format: 'DATE1_DATE2' OR 'DATE1'. The first form declares the first"
           + " date-time (non-inclusive) and last date-time (inclusive); the second form declares"
           + " the active period to be from the given date-time (non-inclusive) until now."
           + " Resources outside the active period are only fetched if they are associated with"
-          + " Patients in the active period. All requested resources in the active period are"
+          + " Patients in the active period . All requested resources in the active period are"
```

Comment:
nit. remove white space between `period` and `.`

Reply:
Done.